src/StableHashTraits.jl

```diff
@@ -456,18 +462,30 @@ end
     stable_typename_id(x)
 
 Returns a 64 bit hash that is the same for a given type so long as the name and the module
-of the type doesn't change. E.g. `stable_typename_id(Vector) == stable_typename_id(Matrix)`
+of the type doesn't change. 
+
+## Example
+
+```jldoctest
+julia> stable_typename_id([1, 2, 3])
+0x56c6b9ca080a0aa4
+
+julia> stable_typename_id(["a", "b"])
+0x56c6b9ca080a0aa4
```

Comment:
now that's interesting! I'm guessing here it's just looking at `AbstractArray` whereas below it knows it's `AbstractArray{Int64, 1}` vs `AbstractArray{String, 1}`?